staging/src/k8s.io/apimachinery/pkg/conversion/unstructured/BUILD

```diff
@@ -3,17 +3,6 @@ package(default_visibility = ["//visibility:public"])
 load(
     "@io_bazel_rules_go//go:def.bzl",
     "go_library",
-    "go_test",
-)
-
-go_test(
-    name = "go_default_test",
-    srcs = ["converter_test.go"],
-    library = ":go_default_library",
-    deps = [
-        "//vendor/k8s.io/apimachinery/pkg/util/diff:go_default_library",
-        "//vendor/k8s.io/apimachinery/pkg/util/json:go_default_library",
-    ],
```

Comment:
I ran `hack/update-bazel.sh` but it did not remove this `go_test` section even though there are no test files in this directory anymore. I removed it myself. Probably a bug in whatever is ran in `update-bazel.sh` script.

Reply:
It seems to be broken.

/cc @mikedanese @frobware